data/governments.txt

```diff
@@ -173,9 +173,12 @@ government "Coalition"
 	"hostile hail" "hostile coalition"
 	"hostile disabled hail" "hostile disabled coalition"
 
+# https://github.com/endless-sky/endless-sky/pull/9436#pullrequestreview-1666400216
```

Comment:
Is linking it like this a good idea? Who is going to want to follow a link to GitHub just to find out why things are the way they are? Also, I believe this would be the only time a comment in the datafiles contains a hyperlink, and one of only eight hyperlinks overall (the rest being all online code examples in Catch.hpp, someone else's library, or build/editor config files).

Overall, it would be better to just have a clear, concise explanation on this line.

Reply:
How about "The Deep government was added before Deep Security and deleting this government could break saves"

Reply:
Sure. Or maybe something like `This is an independent government from Deep Security used for the Deep Archaeology mission change. It was originally added before Deep Security and has been kept to prevent saves being broken.`

Reply:
OK, it now says that instead. Thanks for the input.

Reply:
I don't think a comment is necessary. If anyone wants to track down why something was done, that's what git blame is for.

Reply:
I looked at git blame. At the moment if you look at the government itself, you can see:
-The Deep government appeared in that spot 8 years ago when the governments.txt file was reorganized
-The Deep Security government was added later for one free worlds mission
Looking at the fleets file you can determine that the Deep Security fleets were added shortly after. 
Looking at the names file you can see what was likely an oversight: the deep ships had "D.S.S" names while the ones in deep archaeology had "D.P.S" names. This is likely why they were later called "Deep Security." 
Looking at the Deep Archaeology missions, you can determine that the deep archaeology missions are in the sheragi folder because at the time there was no deep missions file and the rim archaeology missions were also there.  
Going back in time and looking at an old tag, you can determine that when the mission was created, the friendly NPCs were the Republic government and the hostile ones were the Deep government. The friendly ones were likely switched over to Deep Security soon after. In those cases it makes perfect sense: they would want to blend in with everyone else. Making specific republic ships chase after you at a time when there were no display names would mean permanently locking in a " Republic " government for one mission, so this all makes sense. 
With my changes done, git blame will probably explain as much as is necessary, though.
I will remove the comment.